includes/class-akamai.php

```diff
@@ -230,13 +230,13 @@ protected function get_purge_body( $options, $post ) {
 			$this->get_item_url( $permalink ), // Post
 		);
 
-		if ( $options['purge_front'] ) {
+		if ( ! empty( $options['purge_front'] ) && $options['purge_front'] ) {
```

Comment:
@csalzano is this redundant logic?  if `!empty( $options['x'] )` evaluates to false, won't `$options['x']` always be false too?

Reply:
This value corresponds to a checkbox. It will be empty until the user clicks the `Save settings` button for the first time. If not empty, we want to enter the block if the checkbox is checked. The additional clause to check for true is because a zero is saved when the checkbox is left unchecked but the option does exist.

Reply:
I think empty() will still be TRUE if the saved value is `0` (integer) or `'0'` (string), at least based on my reading of https://www.php.net/manual/en/function.empty.php

Reply:
You're right about `purge_front`. I was wrong when I said this setting is corresponding to a checkbox. It's not a setting.

I've used the `purgeOnCommentStatus()` method to capture these warnings rather than `purge_front`. These are the warnings I found and fought off, and I must have though `purge_front` was another one of them for a moment. 

```
[22-Nov-2022 19:05:28 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wp-content/plugins/akamai/includes/class-akamai.php on line 216
[22-Nov-2022 19:05:31 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wp-content/plugins/akamai/includes/class-akamai.php on line 216
```